src/JobsUtilities.py

```diff
@@ -195,3 +197,26 @@ async def getJobs(
             except Exception as e:
                 logging.exception("Failed to process job posting: %s\nJob: %s", e, job)
                 continue
+
+def get_latest_internship_repo():
+    g = Github(GITHUB_TOKEN)
+    org = g.get_organization("SimplifyJobs")
+    repos = org.get_repos()
+
+    matching_repos = []
+    for repo in repos:
+        if repo.name.startswith("Summer"):
+            suffix = repo.name[len("Summer"):]
+            year = suffix.split("-")[0]
+            if len(year) == 4 and year.isdigit():
+                matching_repos.append(repo.name)
+
+    if not matching_repos:
+        raise ValueError(
+            "No repositories matching the pattern 'SummerYYYY-Internships' were found in the organization SimplifyJobs. Make sure the naming format hasn't changed and that your GitHub token has the necessary permissions."
+        )
+
+    latest_repo = max(
+        matching_repos, key=lambda name: int(name[len("Summer"):].split("-")[0])
```

Comment:
Do we still need to sort here? Wouldn't the max return from [2024Internships, 2025Internships] just be 2025Internships?

Reply:
No, this would not include `Internships`. It would just be the year `20XX` part